modules/widgets/authors.php

```diff
@@ -90,14 +90,27 @@ public function widget( $args, $instance ) {
 		 */
 		$excluded_authors = apply_filters( 'jetpack_widget_authors_exclude', $default_excluded_authors );
```

Comment:
I would suggest deprecating this filter, because once we introduce the new one we won't really need that old one anymore.

As a start, I would add a deprecated comment to the docblock, something along the lines of 

`@deprecated 7.7.0 Use jetpack_widget_authors_params instead.`

Then, I would add the deprecated filter and its replacement to the `deprecated_hooks` function in `class.jetpack.php`.

Maybe we can take that opportunity to get rid of `$default_excluded_authors` and directly pass the empty array in there to keep things simple.

```suggestion
		$excluded_authors = apply_filters( 'jetpack_widget_authors_exclude', array() );
```